ui/app/components/mount-backend-form.ts

```diff
@@ -13,7 +13,9 @@ import { methods } from 'vault/helpers/mountable-auth-methods';
 import { isAddonEngine, allEngines } from 'vault/helpers/mountable-secret-engines';
 
 import type FlashMessageService from 'vault/services/flash-messages';
-import type Store from '@ember-data/store';
+import type StoreService from 'vault/services/store';
```

Comment:
`@ember-data/store` doesn't have type definitions included? 🤔

Reply:
Yes that would work since I disabled the eslint rule that complains about importing from `ember-data` rather than `@ember-data`. Since there was already an interface created within our types I just updated it to import the type from `ember-data/store` and then exported that.

Now that you mention it though, I could see this being confusing and I think it's probably worthwhile getting this working from the `@ember-data` namespace.